setup.py

```diff
@@ -10,14 +10,15 @@
 here = os.path.abspath(os.path.dirname(__file__))
 README = open(os.path.join(here, 'README.rst')).read()
 CHANGES = open(os.path.join(here, 'CHANGES.rst')).read()
-REQUIRES_PYTHON = ">=2.7.0"
+REQUIRES_PYTHON = ">=3.6.0"
 
 about = {}
 with open(os.path.join(here, 'emu', '__version__.py'), 'r') as f:
     exec(f.read(), about)
 
 reqs = [line.strip() for line in open('requirements.txt')]
 dev_reqs = [line.strip() for line in open('requirements_dev.txt')]
+gis_reqs = [line.strip() for line in open("requirements_gis.txt")]
```

Comment:
maybe use "extra" instead of "gis" with `requirements_extra.txt`? 

We might add more optional dependencies.